### PR TITLE
[suyabay][#103071990] Hotfix: indentation fixes and directory restructure

### PR DESCRIPTION
**Pull request by fokosun**
… markup code on the master, index, top_nav, side_nav, footer page views to conform with the PSR2 coding standards on indentation, added a new file named modal forms to hold the signin and signup forms, renamed feeds to contents and added a main.blade.php file to it. This holds the contents of the main area of the index page

### CODE REVIEW

File: resources/views/app/includes/contents/main.blade.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+                <div class="card-content">
+                    <span class="card-title activator grey-text text-darken-4">Suya Andela
+                        <i class="material-icons right">view_headline</i>
+                    </span>
+                </div>
+
+                <div class="card-reveal">
+                    <span class="card-title grey-text text-darken-4">Card Title<i class="material-icons right">close</i></span>
+                        <p>Here is some more information about this product that is only revealed once clicked on.</p>
```
Comment (unicodeveloper) [inline resources/views/app/includes/contents/main.blade.php:19]: The paragraph tag doesn't need to be shifted like this, it should still be the on the same grid line with the span tag. Please Fix

File: resources/views/app/includes/contents/main.blade.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                </div>
+
+            </div>
+
+            <div class="card col s12 m4">
+
+                <div class="card-image waves-effect waves-block waves-light">
+                    <img clss = "activator" src="http://a.fastcompany.net/multisite_files/fastcompany/imagecache/slideshow_large/slideshow/2015/05/3046442-slide-s-2-this-startup-pays-african-students.jpg">
```
Comment (unicodeveloper) [inline resources/views/app/includes/contents/main.blade.php]: What's `clss`? ..Where you trying to write `class` ?

File: resources/views/app/includes/contents/main.blade.php [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+                <div class="card-content">
+                    <span class="card-title activator grey-text text-darken-4">Suya Andela
+                        <i class="material-icons right">view_headline</i>
+                    </span>
+                </div>
+
+                <div class="card-reveal">
+                    <span class="card-title grey-text text-darken-4">Card Title
+                        <i class="material-icons right">close</i></span>
```
Comment (unicodeveloper) [inline resources/views/app/includes/contents/main.blade.php]: The `</span>` should be on a line of its own

File: resources/views/app/includes/contents/main.blade.php [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                            <p>Here is some more information about this product that is only revealed once clicked.</p>
+                </div>
+
+            </div>
+
+            <div class="card col s12 m4">
+
+                <div class="card-image waves-effect waves-block waves-light">
+                    <img clss = "activator" src="http://a.fastcompany.net/multisite_files/fastcompany/imagecache/slideshow_large/slideshow/2015/05/3046442-slide-s-2-this-startup-pays-african-students.jpg">
```
Comment (unicodeveloper) [inline resources/views/app/includes/contents/main.blade.php]: `clss` should be `class`

File: resources/views/app/includes/contents/main.blade.php [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+                <div class="card-content">
+                    <span class="card-title activator grey-text text-darken-4">Suya Andela
+                        <i class="material-icons right">view_headline</i>
+                    </span>
+                </div>
+
+                <div class="card-reveal">
+                    <span class="card-title grey-text text-darken-4">Card Title
+                        <i class="material-icons right">close</i></span>
```
Comment (unicodeveloper) [inline resources/views/app/includes/contents/main.blade.php]: `</span>` should be on a line of its own

File: resources/views/app/includes/contents/main.blade.php [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+                            <p>Here is some more information about this product that is only revealed once clicked.</p>
+                </div>
+
+            </div>
+
+            <div class="card col s12 m4">
+
+                <div class="card-image waves-effect waves-block waves-light">
+                    <img clss = "activator" src="http://a.fastcompany.net/multisite_files/fastcompany/imagecache/slideshow_large/slideshow/2015/05/3046442-slide-s-2-this-startup-pays-african-students.jpg">
```
Comment (unicodeveloper) [inline resources/views/app/includes/contents/main.blade.php]: clss should be class

File: resources/views/app/includes/contents/main.blade.php [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+                <div class="card-content">
+                    <span class="card-title activator grey-text text-darken-4">Suya Andela
+                        <i class="material-icons right">view_headline</i>
+                    </span>
+                </div>
+
+                <div class="card-reveal">
+                    <span class="card-title grey-text text-darken-4">Card Title
+                        <i class="material-icons right">close</i></span>
```
Comment (unicodeveloper) [inline resources/views/app/includes/contents/main.blade.php]: `</span>`  should be on a line of its own

File: resources/views/app/includes/contents/main.blade.php [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+                  <p>Here is some more information about this product that is only revealed once clicked on.</p>
+                </div>
+
+            </div>
+
+            <div class="card col s12 m4">
+
+                <div class="card-image waves-effect waves-block waves-light">
+                    <img clss = "activator" src="http://a.fastcompany.net/multisite_files/fastcompany/imagecache/slideshow_large/slideshow/2015/05/3046442-slide-s-2-this-startup-pays-african-students.jpg">
```
Comment (unicodeveloper) [inline resources/views/app/includes/contents/main.blade.php]: clss should be class

File: resources/views/app/includes/contents/main.blade.php [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
```
Comment (unicodeveloper) [inline resources/views/app/includes/contents/main.blade.php]: `clss` should be `class`

File: resources/views/app/includes/contents/main.blade.php [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+                <div class="card-content">
+                    <span class="card-title activator grey-text text-darken-4">Suya Andela
+                        <i class="material-icons right">view_headline</i>
+                    </span>
+                </div>
+
+                <div class="card-reveal">
+                    <span class="card-title grey-text text-darken-4">Card Title
+                        <i class="material-icons right">close</i></span>
+                  <p>Here is some more information about this product that is only revealed once clicked on.</p>
```
Comment (unicodeveloper) [inline resources/views/app/includes/contents/main.blade.php]: The `<p>` tag should be on the same grid line as the `<span>` tag

File: resources/views/app/includes/sections/modalforms.blade.php [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+    <div class="modal-content">
+
+        <div class="row">
+            <form class="col s12">
+                <div class="row">
+                    <div class="input-field col s12">
+                        <i class="material-icons prefix">account_circle</i>
+                            <input id="last_name" type="text" class="validate" placeholder="e.g. snakeMonkeyDuh">
+                                <label for="last_name">Username</label>
+                    </div>
```
Comment (unicodeveloper) [inline resources/views/app/includes/sections/modalforms.blade.php:13]: The `<input>` and `<label>` tag should be on the same grid line as the `<i>` tag

File: resources/views/app/includes/sections/modalforms.blade.php [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+                </div>
+
+                <div class="row">
+                    <div class="input-field col s12">
+                        <i class="material-icons prefix">mail</i>
+                            <input id="email" type="email" class="validate" placeholder="e.g. johnsnow@example.com">
+                                <label for="email">Email</label>
```
Comment (unicodeveloper) [inline resources/views/app/includes/sections/modalforms.blade.php]: The `<input>` and `<label>` tag should be on the same grid line as the `<i>` tag

File: resources/views/app/includes/sections/modalforms.blade.php [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="input-field col s12">
+                        <i class="material-icons prefix">vpn_key</i>
+                            <input id="password" type="password" class="validate" placeholder="********">
+                                <label for="password">Password</label>
```
Comment (unicodeveloper) [inline resources/views/app/includes/sections/modalforms.blade.php]: The `<input>` and `<label>` tag should be on the same grid line as the `<i>` tag

File: resources/views/app/includes/sections/modalforms.blade.php [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+<div id="signin" class="modal">
+    <div class="modal-content">
+        <div class="row">
+            <form class="col s12">
+                <div class="row">
+                    <h2>Hello, sparky!</h2>
+                        <div class="input-field col s12">
+                            <i class="material-icons prefix">account_circle</i>
+                                <input id="last_name" type="text" class="validate" placeholder="e.g. snakeMonkeyDuh">
+                                    <label for="last_name">Username</label>
```
Comment (unicodeveloper) [inline resources/views/app/includes/sections/modalforms.blade.php]: The `<input>` and `<label>` tag should be on the same grid line as the `<i>` tag

File: resources/views/app/includes/contents/main.blade.php (hunk 3 above)
Comment (fokosun) [inline resources/views/app/includes/contents/main.blade.php]: Fixed

File: resources/views/app/includes/sections/modalforms.blade.php [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+                        </div>
+                </div>
+                <div class="row">
+                    <div class="input-field col s12">
+                        <i class="material-icons prefix">vpn_key</i>
+                            <input id="password" type="password" class="validate" placeholder="********">
+                                <label for="password">Password</label>
```
Comment (unicodeveloper) [inline resources/views/app/includes/sections/modalforms.blade.php]: The `<input>` and `<label>` tag should be on the same grid line as the `<i>` tag

File: resources/views/app/includes/sections/modalforms.blade.php [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+                    </div>
+                </div>
+            </form>
+        </div>
+    </div>
+
+    <!-- Signin and forgot password buttons-->
+    <div class="modal-footer">
+        <a href="#reset" class="modal-action modal-trigger modal-close waves-effect waves-white btn-flat">Forgot password?</a>
+        <a href="" class="modal-action modal-close waves-effect waves-white btn">SIGN IN <i class="material-icons right">send</i></a>
```
Comment (unicodeveloper) [inline resources/views/app/includes/sections/modalforms.blade.php]: `href=""` shouldn't be empty , at least put a # there.

File: resources/views/app/includes/sections/modalforms.blade.php [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+<div id="reset" class="modal">
+    <div class="modal-content">
+        <div class="row">
+            <form class="col s12">
+                <div class="row">
+                    <h2>Oops, sparky needs a password reset!</h2>
+                        <div class="input-field col s12">
+                            <i class="material-icons prefix">mail</i>
+                                <input id="email" type="text" class="validate" placeholder="e.g. johnsnow@example.com">
+                                    <label for="email">email</label>
```
Comment (unicodeveloper) [inline resources/views/app/includes/sections/modalforms.blade.php]: The `<input>` and `<label>` tag should be on the same grid line as the `<i>` tag

File: resources/views/app/master.blade.php [hunk 18]
```diff
@@ -1,39 +1,68 @@
...
 
+    </head>
     <body>
 
-    @include('app.includes.sections.head')
+        <!-- Top Nav -->
+        @include('app.includes.sections.top_nav')
+        @include('app.includes.sections.modalforms')
+
+            <div class="row">
```
Comment (unicodeveloper) [inline resources/views/app/master.blade.php]: The div tag should be on the same grid line as the @include's on line 40 and 41

**Pull request merged**